lib/command/init.js

```diff
@@ -107,6 +108,13 @@ module.exports = function (initPath) {
       choices: helpers,
       message: 'What helpers do you want to use?',
     },
+    {
+      type: 'confirm',
+      name: 'installGitHubActions',
+      message: 'Add a GitHub Actions workflow?',
+      default: false,
+      when: (answers) => ['Playwright', 'Puppeteer'].includes(answers.helper) === true,
```

Comment:
I don't think this will work for Puppeteer

Reply:
https://github.com/codeceptjs/CodeceptJS/blob/3.x/.github/workflows/puppeteer.yml

I guess this shall work cause checking this file not any additional libs should be installed for puppeteer.

Reply:
But we have 

```
npx playwright install-deps
```